src/components/HeaderWallet.vue

```diff
@@ -118,6 +120,10 @@ export default {
       metamaskWalletAddress: (state) => state.metamask.metamaskWalletAddress,
       metamaskWalletBalance: (state) => state.metamask.metamaskWalletBalance,
       lastEventData: (state) => state.substrate.lastEventData
+    }),
+
+    ...mapGetters({
+      pair: "substrate/wallet"
```

Comment:
what is this for ? this already computed above as this.wallet

Reply:
updated using this.wallet